nextcord/file.py

```diff
@@ -138,6 +145,16 @@ def __init__(
             self.filename is not None and self.filename.startswith("SPOILER_")
         )
 
+    def __enter__(self) -> File:
+        # Set force_close to true when using context manager
+        # and force_close was not provided to __init__
+        if self.force_close is None:
+            self.force_close = True
+        return self
+
+    def __close__(self, *_) -> None:
```

Comment:
```suggestion
    def __exit__(self, *_) -> None:
```
https://docs.python.org/3/reference/datamodel.html#with-statement-context-managers

Reply:
Yeah, brainfart while copying it from one repo to the other.